2_process/src/diag_func.R

```diff
@@ -0,0 +1,25 @@
+
+diag_func <- function (data,out_dir) {
+  
+  library(dplyr)
+  library(stringr)
+  library(whisker)
```

Comment:
It's a good idea to load libraries _outside_ of functions (so I would suggest just moving this up above `diag_func()`